### ISSUE TITLE
Busted syntax: function brackets in string break highlighting

### ISSUE THREAD

**Issue by tyrondis** [coronalabs/CoronaSDK-SublimeText#20]
If I write a Busted test like this:

```
describe("foo()", function ()
    local bar = 123
end)
```

The syntax highlighting is wrong after the string `foo()`.

![screen shot 2016-03-02 at 3 12 06 am](https://cloud.githubusercontent.com/assets/3440273/13448583/aa7789fe-e024-11e5-8654-d238739387cb.png)

**Comment by perry-clarke** [comment]
That is strange ... something about the combination of the brackets in the string and the inline function definition is making Sublime wig out (and it's the Corona Lua syntax definition; the default Lua syntax definition doesn't have the issue).

You can workaround the problem by adding a newline:

```
describe("foo()", 
    function ()
        local bar = 123
    end)
```

**Comment by perry-clarke** [comment]
This is fixed in release 1.7.0
